### ISSUE TITLE
State property is not defined for Play/Pause button.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1358]
**Actual Result:**
State property is not defined for Play/Pause button.

**Note:**
Same issue reproducible for Mute/Un-mute button and video clips buttons (throughout).

**Expected Result:**
State property should be defined for Play/Pause button.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Start Narrator by pressing "Ctrl + Win + Enter" key.
3. Navigate to 'Type your message' text box present in bottom of screen, type 'help' and press enter key.
4. Navigate to 'Show an audio attachment' button and activate it.
5. Navigate to Play/Pause button and press space bar two-three times to play/pause button.
6. Observe if state property is defined or not.

**User Impact:**
If state property is not defined for any control then it will be very difficult for the user who is dependent on screen reader to understand the purpose of that control.

**Test Environment:**
OS: Windows 10
OS Build: 17763.107
OS Version: 1809
Browser: Edge 

**MAS Reference:**
[MAS 4.2.1 - Object Information (33A)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858})

**Attachment:**
[MAS4.2.1_ObjectInformation.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2583894/MAS4.2.1_ObjectInformation.zip)

**Comment by compulim** [comment]
We did not customize the playback controls from the browser. If this bug repro on Edge, this bug should goes to their bucket.

@corinagum please try it on Chrome, to fulfill our curiosity. :)

**Comment by corinagum** [comment]
Confirmed as external issue by accessibility team

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.2.1
